Automattic/wp-calypso
# Remove unused Pages First View prototype

This PR reverts the functionality added in https://github.com/Automattic/wp-calypso/pull/7171 and https://github.com/Automattic/wp-calypso/pull/7213.

![screen shot 2017-02-09 at 9 26 42 pm](https://cloud.githubusercontent.com/assets/1017839/22801763/7d848330-ef0e-11e6-811f-9ab6fb8362e1.png)

This PR removes the prototype for the Pages First View since nobody is working on it anymore and there are no plans to continue in this direction.

To test:

- Run [this branch](https://calypso.live/?branch=remove/pages-first-view) and verify that there are no regressions in:
  - The Stats First View
  - The Pages section
